code/iaas/allocator/pom.xml

```diff
@@ -28,5 +28,10 @@
             <artifactId>cattle-framework-jooq</artifactId>
```

Comment:
How? no idea how I can do that in Eclipse, can you show me? I checked the pom file, it does not seem so. code/iaas/allocator/pom.xml contains code/iaas/api-logic/pom.xml as dependency, but code/iaas/api-logic/pom.xml does not have any dependency that could lead back to code/iaas/allocator/pom.xml.